app/src/main/java/org/hackillinois/android/view/ScannerFragment.kt

```diff
@@ -61,12 +61,12 @@ class ScannerFragment : Fragment() {
                     val userId: String = getUserIdFromQrString(it.text)
                     viewModel.checkUserIntoEvent(event, userId, view.staffOverrideSwitch.isChecked)
                     Handler(Looper.getMainLooper()).post {
-                        Toast.makeText(context, "Please wait", Toast.LENGTH_LONG).show()
+                        Snackbar.make(view.rootView, "Please wait", Snackbar.LENGTH_LONG ).show()
```

Comment:
I assume the program won’t compile without the space at the end of the function 😅